qemu/tests/x86_cpu_flags.py

```diff
@@ -22,6 +22,13 @@ def run(test, params, env):
     if check_host_flags:
         check_cpu_flags(params, flags, test)
 
+    no_models = params.get("no_models", "")
+    cpu_model = params['cpu_model']
```

Comment:
Not all flag test cases define `cpu_model`, right? Then they might hit error here.

Reply:
> Not all flag test cases define `cpu_model`, right? Then they might hit error here.

Thanks and update, run one other case random and can PASS.